File: translations/codenet/uncommented/Go2C++/codellama/Code/s693332680.cpp
#include <iostream>
using namespace std;

int main() {
	int x, a, b;
	cin >> x >> a >> b;
	switch (b-a) {
		case 0:
			cout << "delicious" << endl;
			break;
		case 1:
			cout << "safe" << endl;
			break;
		default:
			cout << "dangerous" << endl;
			break;
	}
	return 0;
}

File: translations/codenet/gpt_commented/Go2P/granite/Code/s693332680.py

import sys

# Read three integer values from standard input
x, a, b = map(int, sys.stdin.readline().split())

# Evaluate the difference between b and a to determine the output
if b - a <= 0:
    print("delicious") # Output "delicious" if b is less than or equal to a
elif b - a <= x:
    print("safe") # Output "safe" if the difference is within the limit x
else:
    print("dangerous") # Output "dangerous" if the difference exceeds x

